RocketChat/Rocket.Chat
# fix: Thread list not scrolling to bottom when sending a message.

## Proposed changes (including videos or screenshots)






## Issue(s)


## Steps to test or reproduce

When thread content exceeds the viewport's height on the web or desktop, there is no auto-scroll on posting a new message. As a user, if I post a new message in the thread, I then need to manually scroll down to see the message that I posted
## Further comments

TC-534
